module/bdev/lvol/vbdev_lvol.c

```diff
@@ -1709,4 +1709,65 @@ vbdev_lvs_grow(struct spdk_lvol_store *lvs,
 	}
 }
 
+static void
```

Comment:
```
/** Asynchronous event type */
enum spdk_bdev_event_type {
	SPDK_BDEV_EVENT_REMOVE,
	SPDK_BDEV_EVENT_RESIZE,
	SPDK_BDEV_EVENT_MEDIA_MANAGEMENT,
};
```
What events should we handle?
In case of bdev removal we could stop the shallow copy: to do that I think must insert some kind of synchronization that can be be useful for retrieving shallow copy status too

Reply:
I mean, do we need to add a new event type?